features/gba-dev/onbekend_waardes_gba.feature

```diff
@@ -316,7 +316,9 @@ Rule: datumvelden waarde "00000000" worden geleverd
     Gegeven de persoon met burgerservicenummer '000000322' heeft de volgende 'inschrijving' gegevens
     | naam                                             | waarde   |
     | datum opschorting bijhouding (67.10)             | 00000000 |
+    | Europees kiesrecht (31.10)                       | 1        |
```

Comment:
Waarom wordt in de Dan stap geen europeesKiesrecht.aanduiding verwacht?
Idem uitsluitingKiesrecht.uitgeslotenVanKiesrecht?

Reply:
ik zie het al, is niet gevraagd met fields